limetech/lang-es-ES.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Language>
+<LanguageDefault>true</LanguageDefault>
+<Language>Spanish</Language>
+<LanguageLocal>Español</LanguageLocal>
+<LanguagePack>es_ES</LanguagePack>
+<Author>Sergio Fraile, Carlos Rodriguez, Hector Alcala, Brendan Cazier, Christian Sosa, Roy Quiceno</Author>
+<Name>Spanish</Name>
+<LanguageURL>https://github.com/unraid/lang-es_ES</LanguageURL>
+<TemplateURL>https://github.com/limetech/docker-templates/limetech/raw/master/lang-es_ES.xml</TemplateURL>
+<Icon>https://raw.githubusercontent.com/limetech/docker-templates/master/limetech/Green-Earth-Transparent-File.png</Icon>
+<Description>
+This is the built-in Spanish language pack for Unraid.
+</Description>
+</Language>
+© 2020 GitHub, Inc.
```

Comment:
Hey Spencer, you'll want to remove this copyright line

Reply:
ok- will do. Thx Eric